From 95c4e4c5ccafc6d8a948bff0e691904322994ccc Mon Sep 17 00:00:00 2001
From: Kathiravan Thirumoorthy <quic_kathirav@quicinc.com>
Date: Wed, 29 Nov 2023 10:22:36 +0530
Subject: [PATCH] arm64: dts: qcom: remove the IPQ9574 and IPQ5332 dts files

DTS files will be part of qsdk/target/linux/feeds/files-6.1.

Change-Id: I03312736c611859ff13617c014b5174dac6dca0b
Signed-off-by: Kathiravan Thirumoorthy <quic_kathirav@quicinc.com>
---
 .../boot/dts/qcom/ipq5332-rdp-common.dtsi     |  78 --
 1 file changed, 78 deletions(-)
 delete mode 100644 arch/arm64/boot/dts/qcom/ipq5332-rdp-common.dtsi

diff --git a/arch/arm64/boot/dts/qcom/ipq5332-rdp-common.dtsi b/arch/arm64/boot/dts/qcom/ipq5332-rdp-common.dtsi
deleted file mode 100644
index 4870cdb764d0..000000000000
--- a/arch/arm64/boot/dts/qcom/ipq5332-rdp-common.dtsi
+++ /dev/null
@@ -1,78 +0,0 @@
-// SPDX-License-Identifier: BSD-3-Clause
-/*
- * IPQ5332 RDP board common device tree source
- *
- * Copyright (c) 2022-2023 Qualcomm Innovation Center, Inc. All rights reserved.
- */
-
-/dts-v1/;
-
-#include <dt-bindings/gpio/gpio.h>
-#include <dt-bindings/input/input.h>
-#include "ipq5332.dtsi"
-
-/ {
-	aliases {
-		serial0 = &blsp1_uart0;
-	};
-
-	chosen {
-		stdout-path = "serial0";
-	};
-
-	gpio-keys {
-		compatible = "gpio-keys";
-		pinctrl-0 = <&gpio_keys_default>;
-		pinctrl-names = "default";
-
-		button-wps {
-			label = "wps";
-			linux,code = <KEY_WPS_BUTTON>;
-			gpios = <&tlmm 35 GPIO_ACTIVE_LOW>;
-			debounce-interval = <60>;
-		};
-	};
-
-	leds {
-		compatible = "gpio-leds";
-		pinctrl-0 = <&gpio_leds_default>;
-		pinctrl-names = "default";
-
-		led-0 {
-			gpios = <&tlmm 36 GPIO_ACTIVE_HIGH>;
-			linux,default-trigger = "phy0tx";
-			default-state = "off";
-		};
-	};
-};
-
-&blsp1_uart0 {
-	pinctrl-0 = <&serial_0_pins>;
-	pinctrl-names = "default";
-	status = "okay";
-};
-
-&sleep_clk {
-	clock-frequency = <32000>;
-};
-
-&xo_board {
-	clock-frequency = <24000000>;
-};
-
-/* PINCTRL */
-&tlmm {
-	gpio_keys_default: gpio-keys-default-state {
-		pins = "gpio35";
-		function = "gpio";
-		drive-strength = <8>;
-		bias-pull-up;
-	};
-
-	gpio_leds_default: gpio-leds-default-state {
-		pins = "gpio36";
-		function = "gpio";
-		drive-strength = <8>;
-		bias-pull-down;
-	};
-};
-- 
2.34.1

